src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/SwipeControlTests/SwipeControlPage.xaml.cs

```diff
@@ -33,22 +33,21 @@ namespace MUXControlsTestApp
 	[Sample("SwipeControl")]
     public sealed partial class SwipeControlPage : Page //: TestPage
     {
-        object asyncEventReportingLock = new object();
-        List<string> lstAsyncEventMessage = new List<string>();
-        List<string> fullLogs = new List<string>();
-        FrameworkElement lastInteractedWithSwipeControlContentContainer;
-        FrameworkElement lastInteractedWithSwipeControlContentRoot;
-        SwipeItem pastSender;
-        UIElement animatedSwipe;
-        DispatcherTimer _dt;
+        //object asyncEventReportingLock = new object();
+        //List<string> lstAsyncEventMessage = new List<string>();
+        //List<string> fullLogs = new List<string>();
+        //FrameworkElement lastInteractedWithSwipeControlContentContainer;
+        //FrameworkElement lastInteractedWithSwipeControlContentRoot;
+        //SwipeItem pastSender;
+        //UIElement animatedSwipe;
+        //DispatcherTimer _dt;
```

Comment:
Is this needed ?

Reply:
Yes, the compilation was failing on WASM, but this is only a sample control we bring from the the WinUI source code and on which all code behind as already been commented. This fields should also have been commented out.